src/client/Microsoft.Identity.Client/ManagedIdentity/ManagedIdentitySource.cs

```diff
@@ -42,6 +42,11 @@ public enum ManagedIdentitySource
         /// <summary>
         /// The source to acquire token for managed identity is Service Fabric.
         /// </summary>
-        ServiceFabric
+        ServiceFabric,
+
+        /// <summary>
+        /// The source to acquire token for managed identity is Slc Credential Endpoint.
+        /// </summary>
+        SlcCredential
```

Comment:
SLC already contains the word "credential" - maybe `ShortLivedCredential`? I still think we shouldn't expose the name in the API, but this would make it more readable.